.changeset/curly-houses-jump.md

```diff
@@ -0,0 +1,7 @@
+---
+'@emotion/styled': patch
+'@emotion/react': patch
+'emotion-site': patch
```

Comment:
```suggestion
```

We should update Changesets and configure `privatePackages: false` so this doesn't confuse people in the future ;p